experimental/traffic-portal/src/app/core/certs/cert-viewer/cert-viewer.component.html

```diff
@@ -60,7 +60,7 @@
 					</mat-form-field>
 					<mat-form-field>
 						<mat-label>Private Key</mat-label>
-						<textarea matInput name="privateKey" [(ngModel)]="cert.certificate.key" cdkAutosizeMaxRows="15" cdkTextareaAutosize></textarea>
+						<tp-obscured-text-input [name]="'privateKey'" [formControl]="privateKeyFormControl" [disabled]="true" ></tp-obscured-text-input>
```

Comment:
Shouldn't the size (height) of the private key section (able to scroll up and down) be the same as the other two?
<img width="1421" alt="image" src="https://github.com/apache/trafficcontrol/assets/22248619/b192a03f-ffea-4dfb-8ea4-eeeaaa463cb7">